www/package.json

```
@@ -4,9 +4,8 @@
   "repository": "https://gerrit.wikimedia.org/r/apps/ios/wikipedia",
```

Hygiene, don't need to do this if you're not going to make any other changes: Should we update this to our GH repo?

```suggestion
  "repository": "https://github.com/wikimedia/wikipedia-ios/",
```